javaparser-core/src/main/java/com/github/javaparser/printer/configuration/ConfigurablePrinter.java

```diff
@@ -0,0 +1,45 @@
+package com.github.javaparser.printer.configuration;
+
+import java.util.Optional;
+import java.util.Set;
+
+import com.github.javaparser.printer.configuration.PrinterConfiguration.ConfigOption;
+
+/*
+ * This interface defines the api for printer configuration
+ */
+public interface ConfigurablePrinter {
```

Comment:
I've just seen this -- the methods (and the javadoc) here are all related to configuration of a printer (i.e. rename this to `PrinterConfiguration` or similar?)

Reply:
This interface defines what is a configuration of a printer and PrinterConfiguration is an implémentation (default implementation)

Reply:
Changed to interface named PrinterConfiguration and implementation named DefaultPrinterConfiguration.

Reply:
ConfigurablePrinter interface has been renamed in PrinterConfiguration